heudiconv/dicoms.py

```diff
@@ -5,8 +5,7 @@
 from collections import OrderedDict
 import tarfile
 
-import dicom as dcm
-import dcmstack as ds
+from heudiconv.external.pydicom import dcm
```

Comment:
do we have a mandated convention to use relative imports instead of absolute?

I was excited to start using them but then cooled down -- they rarely solve any actual problem, often make it more difficult to actually grasp from where you are importing (e.g. if you have multiple utils.py across different levels of hierarchy, need to count dots, match to the hierarchy -- too much work), and  introduce difficulties during refactoring: e.g. with pycharm I could move functions definitions around and it would anyway convert to absolute imports.

the only time I usually use them is when to import within the same submodule (e.g. `from .utils import ..`) -- there it makes sense to enforce locality.  But for import of the global `lgr` -- not the case

Reply:
yup - that's the case that this handles. if someone wanted to embed heudiconv as a submodule, one should be able to do so.

btw, both . and not dot are called absolute imports. in py3 relative imports are not possible.

Reply:
> to embed heudiconv as a submodule

oh -- should we really aim to make this possible?  IMHO too hypothetical and ATM anyway wouldn't work (we have a mix of import schemes) so I would leave it to be decided and then uniformize one way or another if necessary to achieve the goal.

Reply:
i think every library should be designed with embedding in mind. this is too minor a change to not adopt from the get go. especially for heudiconv which is quite flat already. and other imports are already in local form.

Reply:
hm, ok, let's continue in https://github.com/nipy/heudiconv/issues/170